README.md

```diff
@@ -1,2 +1,17 @@
 phpDocumentor website
 =====================
+
+In order to set up a development environment, you have to have docker and docker-compose.
```

Comment:
Can you put this info in the [github contributors file](https://github.com/blog/1184-contributing-guidelines) and link to that from the README? That will help people on github to find this info.